modules/objc/pages/working-with-vector-search.adoc

```diff
@@ -21,69 +21,65 @@ NOTE: You must install Couchbase Lite to use the Vector Search extension.
 
 This method shows how you can create a vector index using the Couchbase Lite Vector Search extension.
 
-[source, objective-c]
+[source, objc]
 ----
 
-include::objc:example$code_snippets/VectorSearch.m[tags=vs-create-default-config;vs-create-custom-config]
+include::objc:example$code_snippets/VectorSearch.m[tags=vs-create-custom-config]
 
 ----
 
-First, initialize the `config` variable with the `VectorIndexConfiguration()` method with the following parameters:
+First, initialize the `config` object with the `VectorIndexConfiguration()` method with the following parameters:
```

Comment:
A question: how did you find the process of having to make the same changes multiple times for each SDK?

e.g. what's the trade-off with the simpler editing than common page-partials vs redoing the same work in multiple places for you?

Reply:
Good question.